aodntools/timeseries_products/velocity_aggregated_timeseries.py

```diff
@@ -0,0 +1,373 @@
+                    'site_code':                site_code,
+                    'time_coverage_start':      time_start,
+                    'time_coverage_end':        time_end,
+                    'geospatial_vertical_min':  np.min(ds['DEPTH']),
+                    'geospatial_vertical_max':  np.max(ds['DEPTH']),
+                    'geospatial_lat_min':       np.min(ds['LATITUDE']),
+                    'geospatial_lat_max':       np.max(ds['LATITUDE']),
+                    'geospatial_lon_min':       np.min(ds['LONGITUDE']),
+                    'geospatial_lon_max':       np.max(ds['LONGITUDE']),
+                    'date_created':             datetime.utcnow().strftime(timeformat),
+                    'history':                  datetime.utcnow().strftime(timeformat) + ': Aggregated file created.',
+                    'keywords':                 ', '.join(varlist + ['AGGREGATED']),
+                    'rejected_files':           "\n".join(rejected_files),
+                    'contributor_name':        "; ".join(contributor_name),
+                    'contributor_email':       "; ".join(contributor_email),
+                    'contributor_role':        "; ".join(contributor_role),
+                    'generating_code_version':  __version__
+    }
+
+    ## add version
+    github_comment = ('\nThis file was created using https://github.com/aodn/python-aodntools/blob/'
+                      '{v}/aodntools/timeseries_products/aggregated_timeseries.py'.format(v=__version__)
+                      )
+    global_attribute_dictionary['lineage'] += github_comment
+
+    global_attribute_dictionary.update(add_attribute)
+    ds.setncatts(dict(sorted(global_attribute_dictionary.items())))
+
+    ds.close()
+
+
+    ## create the output file name and rename the tmp file
+    facility_code = TStools.get_facility_code(os.path.join(input_dir, files_to_agg[0]))
+    data_code = 'VZ'
+    product_type = 'aggregated-timeseries'
+    file_version = 1
+    output_name = '_'.join(['IMOS', facility_code, data_code, time_start_filename, site_code, ('FV0'+str(file_version)),
+                            ("velocity-"+product_type),
+                            ('END-'+ time_end_filename), 'C-' + datetime.utcnow().strftime(file_timeformat)]) + '.nc'
+    ncout_path = os.path.join(output_dir, output_name)
+    shutil.move(temp_outfile, ncout_path)
+
+
+    return ncout_path, bad_files
+
+
+if __name__ == "__main__":
+
+    parser = argparse.ArgumentParser(description="Concatenate X,Y,Z velocity variables from ALL instruments from ALL deployments from ONE site")
+    parser.add_argument('-site', dest='site_code', help='site code, like NRMMAI',  required=True)
+    parser.add_argument('-files', dest='filenames', help='name of the file that contains the source URLs', required=True)
+    parser.add_argument('-path', dest='output_path', help='path where the result file will be written. Default: ./', default='./', required=False)
```

Comment:
`output_path` is no longer used, so you can remove this line.

Reply:
removed